src/main/resources/db/migration/V8_material_aid_offer.sql

```diff
@@ -0,0 +1,9 @@
+create table material_aid_offer (
```

Comment:
About the filename: according to [flyway docs](https://flywaydb.org/documentation/concepts/migrations.html), version should be separated with two underscores from the name of the script (i.e. `V8__some_name.sql`).